tests/test_fix_flats.py

```diff
@@ -0,0 +1,373 @@
+import pytest
+import numpy as np
+from overflow.fix_flats import (
+    flat_edges,
+    label_flats,
+    away_from_higher,
+    towards_lower,
+    resolve_flats,
+    d8_masked_flow_dirs,
+)
+from overflow.constants import (
+    FLOW_DIRECTION_UNDEFINED,
+    FLOW_DIRECTION_EAST,
+    FLOW_DIRECTION_WEST,
+    FLOW_DIRECTION_SOUTH_EAST,
+    FLOW_DIRECTION_SOUTH_WEST,
+    FLOW_DIRECTION_SOUTH,
+    FLOW_DIRECTION_NORTH_EAST,
+    FLOW_DIRECTION_NORTH_WEST,
+    FLOW_DIRECTION_NORTH,
+)
```

Comment:
Maybe make FLOW_DIRECTION an enum, so you just need to import one thing.

Reply:
Wouldn't this require using `FlowDirection.DIRECTION.value` in each place they are used to get the integer type rather than the Enum type though. Might be easy to accidentally create an array of enums instead of ints or compare and int to an enum. e.g. `Flowdirection.EAST == 1` would be `False` but `FlowDirection.EAST.value == 1` would be `True`

Reply:
Python 3.11 allows enum to be subclasses of other types

https://docs.python.org/3/library/enum.html

```py
from enum import IntEnum
class MyIntEnum(IntEnum):
  A = 1
  B = 2
assert(MyIntEnum.A == 1)
```